apps/netlify/frontend/src/app/netlify-connection.js

```diff
@@ -3,25 +3,30 @@ import PropTypes from 'prop-types';
 import { css } from 'emotion';
 
 import tokens from '@contentful/f36-tokens';
-import { Heading, Paragraph, Icon } from '@contentful/f36-components';
+import { Heading, Paragraph, Button } from '@contentful/f36-components';
+import { LockIcon } from '@contentful/f36-icons';
 
 const styles = {
   auth: css({
     display: 'flex',
     justifyContent: 'center',
   }),
   button: css({
-    backgroundColor: '#00ad9e',
-    color: '#fff',
-    padding: '12px 80px',
-    outline: 'none',
-    borderRadius: '6px',
-    border: '1px solid #e9ebeb',
-    borderBottom: '1px solid #e1e2e4',
-    boxShadow: '0 2px 4px 0 rgba(14,30,37,.12)',
-    cursor: 'pointer',
-    fontSize: '16px',
-    boxSizing: 'border-box',
+    '&&,&&:hover': {
```

Comment:
I am scared

Reply:
It's fine just normal css don't worry about it nothing to see here